client/views/date-change-marker.tpl

```diff
@@ -0,0 +1,3 @@
+<div class="date-change-marker">
```

Comment:
Minor nitpick, can we just call it "date-marker"? No need for the "change" (especially as the first date marker on the screen won't be about the date changing, it's about telling you the original date).